framework/src/main/java/org/checkerframework/framework/qual/TypeUseLocation.java

```diff
@@ -11,6 +11,9 @@
  */
 public enum TypeUseLocation {
 
+    /** Apply default annotations to class and interface declarations. */
```

Comment:
Why not also to enums and annotation types?
Arguably annotation types aren't ever going to be annotated and contain no code.
However, enums contain code and can be annotated.